kirby/api/context.py

```diff
@@ -37,5 +37,8 @@ def __repr__(self):
             ]
         )
 
+    def __getitem__(self, item):
+        return self.__getattr__(item)
```

Comment:
what's the use case for this?

Reply:
I wanted `ctx["<key>"]`  to act as `ctx.<key>`  
It is used to create a Topic (and will be used to WebClient etc...) to specify which variable to look to get the endpoint

Reply:
beware of collisions in this case

Reply:
I will keep that in mind. For this one : Context is just a data class